packages/react-dom/src/shared/ReactControlledValuePropTypes.js

```diff
@@ -36,7 +35,7 @@ if (__DEV__) {
         props.readOnly ||
         props.disabled ||
         props[propName] == null ||
-        enableFlareAPI
+        props.responders
```

Comment:
Can you explain why you removed the `enableFlareAPI` check? Wouldn't this affect someone who isn't using Flare but passes a `responders` prop to an input?

Reply:
Nice find, updated in latest commit.